CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Replace `eslint` with `tslint` as the default linter in new Cloud Functions for Firebase directories. (#1663)
```

Comment:
This should probably say ``Replace `tslint` with `eslint` […]``?

Reply:
Yes, It's a mistake. I fixed this. Thanks!

Reply:
Perfect 👍🏽  🎉